rundeckapp/grails-spa/packages/ui-trellis/src/app/components/job/options/OptionEdit.vue

```diff
@@ -0,0 +1,1402 @@
+          </div>
+          <div class="help-block">
+            {{ $t("form.option.sort.description") }}
+          </div>
+        </div>
+
+        <div
+          class="input-group col-sm-3"
+          :class="{ 'has-error': hasError('delimiter') }"
+        >
+          <div class="input-group-addon" style="background-color: #e0e0e0">
+            {{ $t("form.option.valuesDelimiter.label") }}
+          </div>
+          <input
+            type="text"
+            name="valuesListDelimiter"
+            v-model="option.valuesListDelimiter"
+            size="5"
+            class="form-control"
+          />
+          <div
+            class="help-block"
+            v-if="validationErrors['valuesListDelimiter']"
+          >
+            <ErrorsList :errors="validationErrors['valuesListDelimiter']" />
+          </div>
+        </div>
+        <span class="help-block">
+          {{ $t("form.option.valuesDelimiter.description") }}
+        </span>
+      </div>
+      <div
+        class="form-group opt_keystorage_disabled"
+        :class="{ 'has-error': hasError('regex') }"
+        v-if="!isSecureInput"
+      >
+        <label class="col-sm-2 control-label">{{
+          $t("form.option.enforcedType.label")
+        }}</label>
+        <div class="col-sm-10">
+          <div class="radio">
+            <input
+              type="radio"
+              v-model="enforcedType"
+              value="none"
+              id="enforcedType_none"
+            />
+            <label for="enforcedType_none">
+              {{ $t("none") }}
+              <span class="text-strong">{{
+                $t("form.option.enforcedType.none.label")
+              }}</span>
+            </label>
+          </div>
+          <div class="radio">
+            <input
+              type="radio"
+              v-model="enforcedType"
+              value="enforced"
+              id="enforcedType_enforced"
+            />
+            <label
+              for="enforcedType_enforced"
+              :class="{ 'has-error': hasError('enforced') }"
+            >
+              {{ $t("form.option.enforced.label") }}
+            </label>
+          </div>
+          <div class="radio">
+            <input
+              type="radio"
+              v-model="enforcedType"
+              value="regex"
+              id="etregex_"
+            />
+            <label for="etregex_">
+              {{ $t("form.option.regex.label") }}
+            </label>
+          </div>
+        </div>
+        <div class="col-sm-10 col-sm-offset-2" v-if="enforcedType === 'regex'">
+          <input
+            type="text"
+            name="regex"
+            class="form-control"
+            v-model="option.regex"
+            size="40"
+            :placeholder="$t('form.option.regex.placeholder')"
+            id="vregex_"
+          />
+          <div class="help-block" v-if="validationErrors['regex']">
+            <ErrorsList :errors="validationErrors['regex']" />
```

Comment:
If the default value or the allowed values do not match the regex, the error message isn't showing anywhere. The message keys were 'value' and 'values'

![Screenshot 2024-02-26 at 4 12 21 PM](https://github.com/rundeck/rundeck/assets/139791797/14fc1722-74b6-447f-94e3-eb98162b2fe9)

Reply:
fixed